Resources/Help/preferences/prefsui_nl.md

```diff
@@ -5,7 +5,7 @@ Met de gebruikersinterface parameters in JASP kunnen de volgende opties gespecif
 (Al deze instellingen blijven werken, zelfs na het opnieuw opstarten van JASP.)
 
 ## Lettertype
-Hier kun je aangeven welke lettertype wordt gebruikt door de interface (file menu, ribbon bar en analysisopties paneel), de R, Lavaan of JAGS code, of de analyseresultaat paneel. Als je de standaard lettertype wilt, klik op het 'Standaard lettertype' optie.
+Hier kun je aangeven welke lettertype wordt gebruikt door de interface (file menu, ribbon bar en analysisopties paneel), de R, Lavaan of JAGS code, of de analyseresultaat paneel. Als je geen keuze maakt, de standaard lettertype wordt gebruikt.
```

Comment:
Dit is geen correct nederlands...

"Hier kun je aangeven welk lettertype gebruikt wordt door de interface (bestandsmenu, ribbon balk en de analyseopties), code invoer, of de resultaten. Als je geen keuze maakt, wordt het standaard lettertype gebruikt."